coralproject/talk
# [CORL-1457] Consolidate comment create/replied for Slack events

## What does this PR do?

Allows staff replies to come through on the Slack triggers and post to channels.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Create Slack channel and toggle the staff filter option
- Post as an admin/mod/staff member
- Reply to your post
- See that the Slack channel receives both comments (reply and comment)
